src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -96,6 +96,14 @@ export const ColorPicker = ({
               className="focus-visible-none"
               data-prevent-outside-click
               onCloseAutoFocus={(e) => {
+                // return focus to excalidraw container
+                const container = document.querySelector(
+                  ".excalidraw-container",
+                ) as HTMLElement;
```

Comment:
this won't work for multi-instance scenarios. You can use `useExcalidrawContainer()` hook to get the container.